config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  # resources :users
+  get 'users' => 'users#index'
+  get 'users/:id' => 'users#show'
   get 'dummy' => 'application#dummy_api'
```

Comment:
Once we get this merged in, we should probably remove the dummy endpoint. Would it make sense to make a Trello card so we don't forget about it?